server.bat

```diff
@@ -0,0 +1 @@
+python server.py
```

Comment:
Can we just remove this file and tell folks to run `python server.py` in the README.md?